bidscoin/dicomsort.py

```diff
@@ -96,7 +95,7 @@ def sortsession(sessionfolder: Path, dicomfiles: list, dicomfield: str, rename:
                     else:
                         dicom_fields[field] = value
         if not ext:
-            ext = splitext(dicomfile)[1]
+            ext = Path(dicomfile).suffix()
```

Comment:
That's not what I meant, there is already the `with_suffix` used below

Reply:
But it didn't preserve the extension of the file if ext is absent as it should.